doc/toolbox-enter.1.md

```diff
@@ -40,6 +41,10 @@ host.
 Enter a toolbox container for a different operating system RELEASE than the
 host.
 
+**--once**, **-o**
+
+Do not track container in use, only enter the container once.
```

Comment:
This formulation assumes user already understands the default behavior #135 but it's not yet documented elsewhere.  Consider something more educational like
```suggestion
By default, an escape sequence is printed that some terminal emulators honor, so that new tabs/windows also automatically launch in the toolbox.
After you exit the shell opened directly by this command, Toolbx undoes the escape sequence and you get back to the shell or process that launched `toolbox enter`.

(Effectively, opening a new tab can "copy" the toolbox status from current tab, similar to working directory getting copied.)

This flag disables printing such escape sequences.  It will be up to you to run `toolbox enter` in each terminal where you want it independently.
```

Reply:
I agree. I'll fix it as soon as there is interest in meeting this.